source/ui/SXRMB/SXRMBScanConfigurationView.cpp

```diff
@@ -30,7 +30,7 @@ void SXRMBScanConfigurationView::updateFluorescenceDetectorComboBox(SXRMB::Fluor
 QComboBox *SXRMBScanConfigurationView::createFluorescenceComboBox()
 {
 	QComboBox *newComboBox = new QComboBox;
-	newComboBox->insertItem(0, "None");
+	newComboBox->insertItem(0, "Transmission");
```

Comment:
Hey @XuetaoLiu-CLS, I think it would be better to go with "Choose XRF Detector" and leave it as "None" as the option.  Transmission is a style of XAS where you only use ion chambers, not a detector choice.  In general, the scan is always doing transmission but you optionally have the choice to add an XRF detector.